sdk/maps/azure-maps-examples/src/main/java/com/azure/maps/examples/RenderSample.java

```diff
@@ -68,46 +71,52 @@ public static void main(String[] args) throws IOException {
         builder.addPolicy(subscriptionKeyPolicy);
         RenderClient client = builder.buildClient();
 
```

Comment:
No commented code on PRs please! While OK on my branch, it will NOT be ok on the main branch of the Azure SDK main repository, ever.